opentech/public/funds/blocks.py

```diff
@@ -0,0 +1,22 @@
+from wagtail.wagtailcore.blocks import StaticBlock
+
+from opentech.public.utils.blocks import StoryBlock
+
+
+class ProjectsBlock(StaticBlock):
+    class Meta:
+        icon = 'grip'
+        label = 'List of Projects funded'
```

Comment:
Currently when adding a projects or a reviewers block it shows the default "{label}: this block has no options."

Would be great to add better copy for `admin_text`. Otherwise LGTM!